src/main/java/org/cirdles/topsoil/app/TopsoilMainWindow.java

```diff
@@ -99,8 +104,15 @@ private void initialize() {
                 String.format("%s [%s]", applicationName, applicationVersion));
     }
 
-    Optional<TSVTable> getCurrentTable() {
-        return getCurrentTab().map(tab -> (TSVTable) tab.getContent());
+    Optional<TabContents> getCurrentTable() {
```

Comment:
Should still return `Optional<TSVTable>` unless you feel this method's no longer necessary.